lib/lazy_enumerator.gb

```diff
@@ -74,7 +75,16 @@ class LazyEnumerator
 
   # Returns the first (`size`) elements.
   #
-  def first(size)
+  def first(size = NONE)
```

Comment:
`size = 1` will do the trick without adding extra code :smile:

Reply:
😮 I thought `size = 1` returns `[5]` instead of `5`.

Reply:
I think we still need the logic you added, but can change `size == NONE` to `size == 1`

Reply:
```
irb(main):001:0> [5, 6, 4].lazy.first 1
=> [5]
irb(main):002:0> [5, 6, 4].lazy.first
=> 5
irb(main):004:0> RUBY_VERSION
=> "2.4.2"
```


No argument and 1 will return another value 🤔 Sorry I might be misunderstanding your pointing outs :bow:
I'll re-consider ~